easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.13-foss-2021a-R-4.1.0.eb

```
@@ -0,0 +1,1058 @@
+    }),
+    ('IlluminaHumanMethylation450kanno.ilmn12.hg19', '0.6.0', {
+        'checksums': ['249b8fd62add3c95b5047b597cff0868d26a98862a47cebd656edcd175a73b15'],
+    }),
+    ('IlluminaHumanMethylationEPICanno.ilm10b2.hg19', '0.6.0', {
+        'checksums': ['4decdbc78a6a8d02bf8aecb0d6e1d81134ae9dbc2375add52574f07829e8cd69'],
+    }),
+    ('IlluminaHumanMethylationEPICanno.ilm10b4.hg19', '0.6.0', {
+        'checksums': ['2c8128126b63e7fa805a5f3b02449367dca9c3be3eb5f6300acc718826590719'],
+    }),
+    ('conumee', '1.26.0', {
+        'checksums': ['82f0e066ac038fead0f800c1abc5e2a54ad8b655504b5bc6619627ae7f39ee31'],
+    }),
+    ('BSgenome.Cfamiliaris.UCSC.canFam3', '1.4.0', {
+        'checksums': ['99c55b6f7808822a3dae6679e60ecfb88a2b618159484bc35303c000bd4820c7'],
+    }),
+    ('ExperimentHub', '2.0.0', {
+        'checksums': ['e402f25a540cb68e1c67324fa668beab4c9999edcb1ef1a11fc35100c2e0f5d7'],
+    }),
+    ('SingleR', '1.6.1', {
+        'checksums': ['d66d4c355edf6278616b5164528e651cb3e012aa5c505b49dfe6b53090b51198'],
+    }),
+    ('FlowSorted.Blood.EPIC', '1.10.1', {
+        'checksums': ['7e791e844585b0deca8a879f2c1283cc93590e137441b38ce9f6620ccf68ccb3'],
+    }),
+    ('FlowSorted.CordBloodCombined.450k', '1.8.0', {
+        'checksums': ['b7c78d61f6f2dd823027207bb0ca07b9a4e24c0be12690bb0c46f2852abb7f1d'],
+    }),
+    ('DRIMSeq', '1.20.0', {
+        'checksums': ['a4295a0beb2d0a0717219fa3a7455c9a5fc9e2d5c2e238a7b366133d29695ce7'],
+    }),
+    ('stageR', '1.14.0', {
+        'checksums': ['43a0eec22328571697eb72cd56be56784844b70a5c0841f47a79ab0d9c859cc2'],
+    }),
+    ('isva', '1.9', {
+        'checksums': ['9fd016e0b34034d271d45f8a0d0db62780bf0187112e45f610aa9237014e1d17'],
+    }),
+    ('org.Mm.eg.db', '3.13.0', {
+        'checksums': ['94fcdf89df0f1174eb92327cfc3918f7a1b62be09240926b90b423dcae19ffcb'],
+    }),
+    ('org.Rn.eg.db', '3.13.0', {
+        'checksums': ['748382f57ea49cc06af44bc89e47443c9694554a9657bd567ee8c982d0111dfd'],
+    }),
+    ('ROC', '1.68.0', {
+        'checksums': ['4c6640e9f711bc07f03a3be5b0f55cc30a50418d542bb89cfef22def2fcbc539'],
+    }),
+    ('wateRmelon', '1.36.0', {
+        'checksums': ['fc1f6197df7342b0061e7c77261bdb9622ec1678916b8448c66b0c67c14759e1'],
+    }),
+    ('GLAD', '2.56.0', {
+        'checksums': ['1deeb411fea9a240e79989ae8d84f18e218c2d36f4d7f2b919386ae35276d741'],
+    }),
+    ('missMethyl', '1.26.0', {
+        'checksums': ['f1204a2b1d57a35f823cc7ceb4d6547a1811a914a43390023e51df94208fbfea'],
+    }),
+    ('MethylSeekR', '1.32.0', {
+        'checksums': ['b4268bfd304f5362771e927faf4827ff9b0b9b08f1901b390e68787bd95a3e5a'],
+    }),
+    ('affxparser', '1.64.0', {
+        'checksums': ['3480b9cf9b73240630dba3fa29bde1546324e7c62f26da3107fce34f5ff67e58'],
+    }),
+    ('ccdata', '1.18.0', {
+        'checksums': ['de24c7205723067599de8b7dbec3c8488c9fcf8cbac7948966a82c22bab4ea6d'],
```

@branfosj usual checksum difficulty? My test shows `48c5b1c054972b5ad720e3cb3ad9fe02b0a9fe39590681450c2dbadecaea1e95`

```suggestion
        'checksums': ['48c5b1c054972b5ad720e3cb3ad9fe02b0a9fe39590681450c2dbadecaea1e95],
```

---

I've added it as an alternative in 836025b (and fixed the missing `,` in 704e43d).